zerver/views/auth.py

```diff
@@ -101,7 +101,7 @@ def maybe_send_to_registration(request: HttpRequest, email: str, full_name: str=
             realm = get_realm(get_subdomain(request))
         except Realm.DoesNotExist:
             realm = None
-        streams_to_subscribe = None
+        streams_to_subscribe = []
```

Comment:
I think the right fix for this and the previous commit is instead to do 1655aabe65a4be8b8941085a38e5f9c704d3718a (moving this variable closer to where it's used).  I've merged that.